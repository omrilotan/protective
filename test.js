/* eslint-env jest */

import * as modules from './index.js'

describe('protective', () => {
  test('exposes all functions', () => {
    expect(Object.keys(modules)).toMatchSnapshot()
  })
})
