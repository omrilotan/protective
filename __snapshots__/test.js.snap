// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`protective exposes all functions 1`] = `
Array [
  "stringify",
  "verbose",
]
`;
